craft_providers/multipass/multipass.py

```diff
@@ -128,6 +133,20 @@ def info(self, *, instance_name: str) -> Dict[str, Any]:
 
         return json.loads(proc.stdout)
 
+    def is_supported_version(self) -> bool:
+        """Check if Multipass version is supported.
+
+        A helper to check if Multipass meets minimum supported version for
+        craft-providers.
+
+        :returns: True if installed version is supported.
+        """
+        version, _ = self.version()
+
+        return pkg_resources.parse_version(version) >= pkg_resources.parse_version(
+            self.minimum_required_version
+        )
```

Comment:
Do we have any guarantee that Multipass versions will comply with [PEP 440](https://www.python.org/dev/peps/pep-0440/)? If not, and if it follows semanic versioning, we can use [semantic-version](https://pypi.org/project/semantic-version/) instead.

Looking at past version numbering used in Multipass it seems that it uses plain `major.minor.release` so probably any of these will work. (There was an -rc version but I don't know if that appears in the actual version string.)